pkg/handler3/handler.go

```diff
@@ -49,9 +49,12 @@ const (
 	subTypeJSON     = "json"
 )
 
+type OpenAPIV3Discovery struct {
```

Comment:
needs documentation on type and fields

Reply:
I think you could have split the type docs into type docs and per field docs, but you don't have to change that.

Reply:
Do we need the json tags on the fields? (here and below?)